internal/service/networkpeering/resource_network_peering.go

```diff
@@ -103,21 +103,25 @@ func Resource() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				ForceNew: true,
 			},
 			"azure_subscription_id": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				ForceNew: true,
 			},
 			"resource_group_name": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				ForceNew: true,
 			},
 			"vnet_name": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				ForceNew: true,
```

Comment:
I am not clear why we are marking as `ForceNew` (applies to above attributes as well). My understanding is that after adjusting the PATCH request this property can be updated successfully. Defining `ForceNew` forces terraform to avoid calling the update and instead call delete -> create.

Reply:
When a network peering connection has been created and the status is ACTIVE, the network peering connection can not be modified and instead needs to be deleted and recreated. A network peering connection can only be updated if the creation has failed and the status is FAILED.

Reply:
thanks for clarifying! Would update the changelog entry to clarify that azure properties will force a delete and create.